packages/smooth_app/android/app/src/main/kotlin/org/openfoodfacts/app/smooth_app/MainActivity.kt

```diff
@@ -1,4 +1,4 @@
-package org.openfoodfacts.smooth_app
+package org.openfoodfacts.app
```

Comment:
I'm not sure it's a good idea: I believe that the package name is the ID for Android - but perhaps this is overridden somewhere else? (e.g. in `build.gradle`)

Reply:
.app is actually the real identifier which we have specified everywhere else. This .smooth_app somehow snuck in and was the root cause for most (when not all) android problems. 

Somehow flutter wants to generate it to be .smooth_app but only in this file, it probably takes the folder name where the project is in to guess a name.